app/src/androidTest/java/org/simple/clinic/bp/BloodPressureRepositoryAndroidTest.kt

```diff
@@ -218,4 +218,39 @@ class BloodPressureRepositoryAndroidTest {
     assertThat(repository.bloodPressureCount(patientUuidWithOnlyDeletedBloodPressures).blockingFirst()).isEqualTo(0)
     assertThat(repository.bloodPressureCount(patientUuidWithBloodPressures).blockingFirst()).isEqualTo(1)
   }
+
+  @Test
+  fun querying_whether_blood_pressures_for_patient_have_change_should_work_as_expected() {
+    val patientUuid = UUID.randomUUID()
+    val bpUpdatedAt = Instant.now(clock)
+    val oneMinute = Duration.ofMinutes(1)
+    val twoMinutes = Duration.ofMinutes(2)
+
+    val bp1ForPatient = testData.bloodPressureMeasurement(
+        patientUuid = patientUuid,
+        syncStatus = SyncStatus.PENDING,
+        updatedAt = bpUpdatedAt
+    )
+    val bp2ForPatient = testData.bloodPressureMeasurement(
+        patientUuid = patientUuid,
+        syncStatus = SyncStatus.PENDING,
+        updatedAt = bpUpdatedAt.plus(oneMinute)
+    )
+    val bpForSomeOtherPatient = testData.bloodPressureMeasurement(
+        patientUuid = UUID.randomUUID(),
+        syncStatus = SyncStatus.PENDING,
```

Comment:
We should add more test cases with different `syncStatus` and multiple BPs for a patient with only one BP that has changed since.

Reply:
Done